TLM/TLM/UI/Localization/LookupTable.cs

```diff
@@ -58,7 +58,7 @@ public class LookupTable {
             return AllLanguages[Translation.DEFAULT_LANGUAGE_CODE]
                        .TryGetValue(key, out string ret2)
                        ? ret2
-                       : (VersionUtil.BRANCH != "STABLE")
+                       : (VersionUtil.BRANCH != "STABLE" && VersionUtil.BRANCH != "TEST")
                             ? "¶" + key
                             : key.IndexOf(":") > 0
```

Comment:
This will calculate IndexOf twice. Not nice

Reply:
not my code, ask @aubergine10 😂

Reply:
There's bool value that can be used here: `VersionUtil.IS_DEBUG`

Reply:
i'll send commit with tweaks

Reply:
commit sent - should be cleaner now, including removal of duplicate IndexOf